inter/block_test.go

```diff
@@ -0,0 +1,68 @@
+package inter
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"github.com/Fantom-foundation/lachesis-base/hash"
+	"github.com/ethereum/go-ethereum/common"
+	"math/rand"
+	"testing"
+)
+
+func TestBlock_ComputePrevRandao_ComputationIsDeterministic(t *testing.T) {
+	events := []hash.Event{
+		{byte(rand.Int())},
+		{byte(rand.Int())},
+		{byte(rand.Int())},
+		{byte(rand.Int())},
+	}
+	blk := Block{Events: events}
+	randao1 := blk.GetPrevRandao()
+	rand.Shuffle(len(blk.Events), func(i, j int) {
+		blk.Events[i], blk.Events[j] = blk.Events[j], blk.Events[i]
+	})
+	randao2 := blk.GetPrevRandao()
+	if randao1 != randao2 {
+		t.Error("computation is not deterministic")
+	}
+}
+
+func TestBlock_ComputePrevRandao_ComputationWorks(t *testing.T) {
+	tests := []struct {
+		name   string
+		events hash.Events
+	}{
+		{
+			name:   "empty_events",
+			events: []hash.Event{},
+		},
+		{
+			name: "one_events",
+			events: []hash.Event{
+				{byte(rand.Int())},
+			},
+		},
+		{
+			name: "multiple_events",
+			events: []hash.Event{
+				{byte(rand.Int())},
+				{byte(rand.Int())},
+				{byte(rand.Int())},
```

Comment:
Please use Event IDs with more than a single byte. Position 0 filled here is anyway ignored in the computation, since only positions 8-31 are used in the PrevRandao computaiton.